charts/operator/deployment.go

```diff
@@ -355,11 +316,18 @@ func operatorArguments(dot *helmette.Dot) []string {
 
 	args := []string{
 		"--health-probe-bind-address=:8081",
-		"--metrics-bind-address=127.0.0.1:8080",
+		"--metrics-bind-address=:8443",
 		"--leader-elect",
 		fmt.Sprintf("--webhook-enabled=%t", isWebhookEnabled(dot)),
 	}
 
+	if isWebhookEnabled(dot) {
+		args = append(args,
+			"--webhook-enabled=true",
+			"--webhook-cert-path=/tmp/k8s-webhook-server/serving-certs",
```

Comment:
This is a very magical looking string, where is it coming from?

Reply:
It comes from https://github.com/redpanda-data/redpanda-operator/blob/054413d40605338549a4fd85786a817599d7fa19/charts/operator/deployment.go#L346